lib/src/presentation/scenes/home/home_screen_bloc.dart

```diff
@@ -67,7 +71,10 @@ class HomeScreenBloc {
     _logoutStateSubject.sink.add(result);
   }
 
-  void selectHabit(Habit selectedHabit) {
+  void toggleHabit(Habit selectedHabit) {
+    selectedHabit.isSelected =
+        selectedHabit.isSelected == null ? true : !selectedHabit.isSelected;
+    selectedHabits.add(selectedHabit);
```

Comment:
@phenybutum here also must be removing habit from selected habits list when unselect habit